core/src/main/scala/io/scalac/mesmer/core/module/Module.scala

```diff
@@ -27,36 +22,32 @@ object Module {
     def combine(other: M[T])(implicit combine: Combine[M[T]]): M[T]          = combine.combine(value, other)
     def exists(check: T => Boolean)(implicit traverse: Traverse[M]): Boolean = traverse.sequence(value).exists(check)
   }
-
-  trait CommonJars[T] {
-    def akkaActor: T
-    def akkaActorTyped: T
-  }
 }
 
 trait MesmerModule extends Module with MesmerConfigurationBase {
-  override type Result = Config
+  override type Result = Config with Product
+
+  lazy val enabled: Config = {
```

Comment:
I'n not sure about the following: 
 - is `lazy` needed here?
 - why is it called `enabled`?  Should it be called `enable` instead (verbs indicate action)?